docs/mutations.md

```diff
@@ -37,133 +38,233 @@ any outstanding TODOs before opening issues.
 
 ## Quick reference
 
-```{eval-rst}
-.. todo:: 
-  These summaries need work. We can say more then "amino acids" e.g. for PAM
-```
 
 {func}`.sim_mutations`
-: Simulate mutations on ancestral topology  
+: Add simulated mutations to a tree sequence
 
 **Models**
 
 {class}`.BinaryMutationModel`             
-: 0/1 flip-flopping alleles                 
+: Binary mutation model with two flip-flopping alleles: "0" and "1".
 
 {class}`.JC69MutationModel`               
-: Jukes & Cantor '69, nucleotides           
+: Jukes & Cantor model ('69), equal probability of transitions between nucleotides
 
 {class}`.HKYMutationModel`                
-: Hasegawa, Kishino & Yano '85, nucleotides 
+: Hasegawa, Kishino & Yano model ('85), different probabilities for transitions and transversions
 
 {class}`.F84MutationModel`                
-: Felsenstein '84, nucleotides              
+: Felsenstein model ('84), different probabilities for transitions and transversions
 
 {class}`.GTRMutationModel`                
-: general time-reversible, nucleotides      
+: Generalised Time-Reversible nucleotide mutation model
 
-{class}`.BLOSUM62MutationModel` 
-: amino acids                               
+{class}`.BLOSUM62MutationModel`                
+: The BLOSUM62 model of time-reversible amino acid mutation
 
 {class}`.PAMMutationModel`              
-: amino acids                               
+: The PAM model of time-reversible amino acid mutation
 
 {class}`.MatrixMutationModel`             
-: general finite-state mutations            
+: Superclass of mutation models with a finite set of states
 
 {class}`.InfiniteAllelesMutationModel`    
-: a generic infinite-alleles model          
+: A generic infinite-alleles mutation model          
 
 {class}`.SLiMMutationModel`               
-: SLiM compatible mutations                 
+: An infinite-alleles model producing SLiM-style mutations
 
 ---
 
-```{eval-rst}
-
-.. todo:: 
-    This has been copied directly from the api.rst file. Probably
-    needs some refactoring to use the newer docs arrangements.
-    Also all the labels need updating.
-```
+{func}`.sim_mutations` adds mutations to a tree sequence simulated with
```

Comment:
Not necessarily limited to those generated by sim_ancestry, maybe "or any other program that outputs tree sequence topologies".